### PR TITLE
Version bump for #24 and #26

### PR DESCRIPTION
**Pull request by webmat**
Version bump for #24 and #26

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
+## 3.0.5
+  - Get rid of a Ruby warning from using `timeout`.
+  - Reduce log noise at info level, there shouldn't be a log for every single.
+  - Docs: Better directions for getting a key in Loggly & other cleanups.
+  - Get rid of a few `puts` and reduce the noise at logging level `info`.
+    There is no longer 1+ log/stdout line created for every event ingested.
+
```
Comment (robbavey) [inline CHANGELOG.md]: Nit:  Can you add a reference to the PR that solves each of these (adding #24 to `Get rid of a Ruby warning...`, for example)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
 ## 3.0.4
-  - Docs: Set the default_codec doc attribute.
+  - Docs: Set the default\_codec doc attribute.
```
Comment (robbavey) [inline CHANGELOG.md]: nice catch

File: CHANGELOG.md (hunk 1 above)
Comment (webmat) [reply CHANGELOG.md]: Good idea

Comment (webmat) [reply CHANGELOG.md]: So, line 1 is PR #24. All other lines are #26. I'll try a way to format this, let me know your thoughts

Comment (webmat) [reply CHANGELOG.md]: @robbavey Do you think this looks wonky? https://github.com/webmat/logstash-output-loggly/blob/3468e8f080ca18c74ef92df6c20e6d52bafc69d3/CHANGELOG.md

Is this how it's usually done? Or should I just suck it up and add #26 on each line ;-)

Comment (robbavey) [reply CHANGELOG.md]: @webmat We don't have a standardized changelog format yet. I've typically added the link at the end, although I've not had as many changes rolled into one PR as this one ;)

Comment (webmat) [reply CHANGELOG.md]: Ok, I thought this looked bad. There's now a full #26 link at the beginning of each line. The code looks repetitive, but the rendered version looks much better.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,19 @@
+## 3.0.5
+  - [#24](https://github.com/logstash-plugins/logstash-output-loggly/pull/24)
+    Get rid of a Ruby warning from using `timeout`.
+  - [#26](https://github.com/logstash-plugins/logstash-output-loggly/pull/26)
+    Docs: Better directions for getting a key in Loggly & other cleanups.
+  - [#26](https://github.com/logstash-plugins/logstash-output-loggly/pull/26)
+    Get rid of a few `puts` and reduce the noise at logging level `info`.
+    There is no longer 1+ log/stdout line created for every event ingested.
+
```
Comment (robbavey) [inline CHANGELOG.md]: 👍

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,10 +1,19 @@
...
 ## 3.0.4
-  - Docs: Set the default_codec doc attribute.
+  - Docs: Set the default\_codec doc attribute.
 
 ## 3.0.3
   - Update gemspec summary
 
-## 3.0.2
+## 3.0.12
```
Comment (robbavey) [inline CHANGELOG.md]: I'm guessing that this was not intentional

Comment (webmat) [reply CHANGELOG.md]: Wow, wonder how vim thought I wanted to do that. Good catch.

**Comment by webmat** [comment]
I definitely will. Thanks!

**Pull request closed without merging**